soc/arm/ambiq/apollo4x/soc.c

```diff
@@ -10,7 +10,14 @@
 
 static int arm_apollo4_init(void)
 {
+
+	/* Initialize for low power in the power control block */
 	am_hal_pwrctrl_low_power_init();
+
+	/* Enable SIMOBUCK for this board */
+	am_hal_pwrctrl_control(AM_HAL_PWRCTRL_CONTROL_SIMOBUCK_INIT, 0);
```

Comment:
The comment suggests this is board-specific, but you're doing it unconditionally at the SoC level. Do you need to add a Kconfig to allow different boards to enable/disable the feature?

Reply:
@MaureenHelm good catch! It is chip specific and not board specific.
I will change the comment.